README.md

```diff
@@ -158,34 +152,83 @@ An export configuration entry in the `export` arrays contains the following keys
 * `strictTypeExport`: If a resource has multiple types and these extras should not be exported, set this to `true`
 
 The `export` array may contain multiple entries for an `rdf:type`.
-A resource may have multiple types and therefore map to multiple export configugrations, but the `pathToConceptScheme` must be identical in that case. The service only works under the assumption that a resource has only 1 way to the export concept scheme (but there may be multiple instances of that way).
+A resource may have multiple types and therefore map to multiple export configurations, but the `pathToConceptScheme` must be identical in that case. The service only works under the assumption that a resource has only 1 way to the export concept scheme (but there may be multiple instances of that way).
+
+#### Deltastream configuration
+The deltastream configuration allows for multiple deltastreams in one instance, it is specified by `CONFIG_SERVICES_JSON_PATH`.
+The json file represents a dictionary of service configurations with the key being the name of the service and the value the configuration.
+The proporties are:
+* `jobsGraph (default: "http://mu.semte.ch/graphs/system/jobs")`: URI where the jobs and jobs information are stored
+* `errorCreatorUri (default: "http://lblod.data.gift/services/delta-producer-publication-graph-maintainer")`: URI of the creator of errors
+* `healingPatchGraphBatchSize (default: 100)`: Size of insert/delete batches.
+* `updatePublicationGraphSleep (default: 1000)`: Sleep between batch updates in the update of the publication graph
+* `muCallScopeIdPublicationGraphMaintenance (default: http://redpencil.data.gift/id/concept/muScope/deltas/publicationGraphMaintenance)`: can be configured to work with scopes in delta-notifier. This is fired when a updates are performed on the publication graph. Most of the services in your stack wouldn't be interested by this update, so best to add this in the deltanotifier configuration as scope to exclude. So we reduce load on the system, and potential confusion.
+* `muCallScopeIdInitialSync (default: http://redpencil.data.gift/id/concept/muScope/deltas/initialSync)` : can be configured to work with scopes in delta-notifier. This when the publication graph is initially synced. Most of the services in your stack (if not all) wouldn't be interested by this update, so best to add this in the deltanotifier configuration as scope to exclude.
+* `waitForInitialSync`: wait for initial sync. Defaults to 'true', mainly meant to disable for debugging purposes
+* `publicationGraph (required)`: URI of the publication graph to maintain
```

Comment:
<s>when testing app-lblod-harvester, I got  an error `Expected 'PUBLICATION_GRAPH' should be provided`, I think it has to be updated in the config</s>
Seems the image was not updated after commit